CI/check-services.py

```diff
@@ -58,6 +59,12 @@
 
 context = ssl.create_default_context()
 
+def print_public_ip():
+    try:
+        resp = requests.get(PUBLIC_IP_ENDPOINT)
```

Comment:
Maybe also add the timeout here like with the HLS check, just to avoid it getting stuck for a long time.